modules/andBeyondMediaBidAdapter.js

```diff
@@ -0,0 +1,202 @@
+import { isFn, deepAccess, logMessage, logError } from '../src/utils.js';
+
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER, NATIVE, VIDEO } from '../src/mediaTypes.js';
+import { config } from '../src/config.js';
+
+const BIDDER_CODE = 'andbeyondmedia';
```

Comment:
The bidder code should be unique for the first 6 characters. The `adkernelBidAdapter` adapter uses the bidder code alias: `andbeyond`. You may want to consider changing the name a little to avoid confusion.

Reply:
thanks, changed